src/reducers/app.js

```diff
@@ -0,0 +1,39 @@
+import actionTypes from '../constants/actionTypes';
```

Comment:
Same here, maybe name it `participants`?

Reply:
got it